src/tests/process/dhcpv6/Request

```diff
@@ -0,0 +1,63 @@
+subrequest DHCPv6.Request {
+	&Transaction-ID = 0x1e291d
+
+	&Server-ID.DUID = LLT
+	&Server-ID.DUID.LLT.Hardware-Type = Ethernet
+	&Server-ID.DUID.LLT.Time = "Nov 21 2012 08:36:00 UTC"
+	&Server-ID.DUID.LLT.Hardware-Type.Ethernet.Address = 00:11:22:33:44:55
+
+	&Client-ID.DUID = LL
+	&Client-ID.DUID.LL.Hardware-Type = Ethernet
+	&Client-ID.DUID.LL.Hardware-Type.Ethernet.Address = 00:01:02:03:04:05
+
+	&request += {
+		&Option-Request = DNS-Servers
+		&Option-Request = AFTR-Name
+	}
+
+	&Elapsed-Time = 0
+
+	call dhcpv6 {
+		# 'Request' expects 'Reply'
+		if (&reply.Packet-Type != Reply) {
+			test_fail
+		}
+
+		# We should reply the below attributes...
+		if (!&reply.Transaction-ID) {
```

Comment:
Most of these checks could go into a policy?

```
policy dhcpv6_reply_matches_request {
   ...
```

and then the individual test just runs the policy.

That should simplify things a bit.

I think once that's done, we can merge this